### PR TITLE
Feature website initial cleanup 3

### PR DESCRIPTION
**Pull request by Rene2mt**
Draft PR for issues listed in - https://github.com/GSA/automate.fedramp.gov/issues/21

### CODE REVIEW

**Comment by Rene2mt** [comment]
This PR addresses core (high priority) parts of issues #21.

File: content/about/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: FedRAMP's Use of OSCAL
+title: About FedRAMP OSCAL
```
Comment (david-waltermire) [inline content/about/_index.md:2]: OSCAL is not FedRAMP's. This is about how FedRAMP is using OSCAL.
```suggestion
title: About FedRAMP's Use of OSCAL
```

File: content/documentation/_index.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ or JSON. If necessary, FedRAMP\'s tools will convert the files for
...
 
 {{<callout>}}
-_For more information on converting OSCAL files between XML and JSON, see Section 1.6.2, NIST OSCAL Format Conversion Mechanisms._
+_For more information on converting OSCAL files between XML and JSON, see [NIST OSCAL Format Conversion Mechanisms](/documentation/#nist-oscal-format-conversion-mechanisms)._
 {{</callout>}}
 
 NOTE: NIST partially supports _YAML_ (YAML) as an offshoot of JSON.
 FedRAMP will evaluate the use of YAML for FedRAMP deliverables once NIST
 offers the same level of support for YAML syntax validation and format
 conversion.
```
Comment (david-waltermire) [inline content/documentation/_index.md:61]: YAML is just as supported as JSON and XML. This text needs to be revised.

File: content/documentation/_index.md [hunk 3]
```diff
@@ -105,30 +98,17 @@ Validating JSON-based OSCAL files using the NIST-published schema
 validation requires:\
 [JSON Schema, draft-07](https://json-schema.org/specification-links#23draft-7)
 
-There are several open-source and commercial tools that will process XSD 1.1 or JSON Schema, draft-07, either as stand-alone capabilities or as programming libraries. FedRAMP and NIST are unable to endorse specific products.
+The [NIST OSCAL Build tools](https://github.com/usnistgov/OSCAL/blob/v1.1.2/build/README.md#schemas) provide a mechanism for validating OSCAL XML and JSON content. There are several open-source and commercial tools that will process XSD 1.1 or JSON Schema, draft-07, either as stand-alone capabilities or as programming libraries. FedRAMP and NIST are unable to endorse specific products.
```
Comment (david-waltermire) [inline content/documentation/_index.md:101]: Linking to a specifc version of OSCAL will not stand the test of time. Perhaps we can find a better, perhaps web-based, referemce.

File: content/documentation/general-concepts/oscal-data-types.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+types that directly impact FedRAMP content in OSCAL.
+
+## Date and Time in OSCAL Files
+
+Except where noted, all dates and times in the OSCAL-based content must
+be in an OSCAL
+`dateTime-with-timezone` format as documented here:
+
+[[https://pages.nist.gov/OSCAL/reference/datatypes/#datetime-with-timezone]](https://pages.nist.gov/OSCAL/reference/datatypes/#datetime-with-timezone)
+
```
Comment (david-waltermire) [inline content/documentation/general-concepts/oscal-data-types.md:21]: Please search for these and replace with the appropriate metaschema data type references?

**Review by david-waltermire — CHANGES_REQUESTED** [review]
References to OSCAL data types need to be replaced with the correct pages.nistgov/metaschema data type reference.

We should talk about how to reference specific OSCAL versions.

File: content/documentation/general-concepts/oscal-data-types.md (hunk 4 above)
Comment (Rene2mt) [reply content/documentation/general-concepts/oscal-data-types.md]: Fixed in commit [a7d1f7e](https://github.com/GSA/automate.fedramp.gov/pull/24/commits/a7d1f7eaf05b39442543ce8691b31d049f19177f)

File: content/documentation/_index.md (hunk 2 above)
Comment (Rene2mt) [reply content/documentation/_index.md]: Revised section noting FedRAMP support of YAML in commit [b7d2ca6](https://github.com/GSA/automate.fedramp.gov/pull/24/commits/b7d2ca6a629e104e97d5ae4365af962695670962)

File: content/documentation/_index.md (hunk 3 above)
Comment (Rene2mt) [reply content/documentation/_index.md:93]: We could change this to one of the following:
- link refencing section on GitHub main branch - https://github.com/usnistgov/OSCAL/blob/main/build/README.md#schemas
- Permalink referencing section in GitHub - https://github.com/usnistgov/OSCAL/blob/4f02dac6f698efda387cc5f55bc99581eaf494b6/build/README.md#schemas

OR we could update the links reference the following NIST pages:
- for validation - https://pages.nist.gov/OSCAL/resources/tools/#validation 
- for conversion - https://pages.nist.gov/OSCAL/resources/tools/#data-conversion

Comment (david-waltermire) [reply content/documentation/_index.md:93]: Please use the NIST pages.

Comment (Rene2mt) [reply content/documentation/_index.md]: Fixed in commit [edf56ce](https://github.com/GSA/automate.fedramp.gov/pull/24/commits/edf56ce07c4b253bf2afbb8c1382195425bda7d7)

**Comment by Rene2mt** [comment]
Resolved review comments and suggestions, with exception of how to reference specific OSCAL versions.  Created issue #26 to address how to reference specific OSCAL versions of resources in various OSCAL related GitHub repositories.

**Review by david-waltermire — APPROVED** [review]

**Pull request merged**